test/commons/text/is-human-interpretable.js

```diff
@@ -1,63 +1,71 @@
 describe('text.isHumanInterpretable', function () {
   it('returns 0 when given string is empty', function () {
-    var actual = axe.commons.text.isHumanInterpretable('');
+    const actual = axe.commons.text.isHumanInterpretable('');
     assert.equal(actual, 0);
   });
 
-  it('returns 0 when given string is a single character that is blacklisted as icon', function () {
-    var blacklistedIcons = ['x', 'i'];
-    blacklistedIcons.forEach(function (iconText) {
-      var actual = axe.commons.text.isHumanInterpretable(iconText);
+  it('returns 0 when given string is a single alpha character', function () {
+    const singleCharacterExamples = ['i', '×'];
+    singleCharacterExamples.forEach(function (characterExample) {
+      const actual = axe.commons.text.isHumanInterpretable(characterExample);
+      assert.equal(actual, 0);
+    });
+  });
+
+  it('returns 0 when given string is in the symbolic text characters set (blocklist)', function () {
```

Comment:
You'll want a companion test to this that verifies that capitalization  matters, so `AA` or `AbC` won't match the blocklist.

Reply:
We decided it does not matter after all, still enhanced my tests in [30d8388](https://github.com/dequelabs/axe-core/pull/4402/commits/30d8388976118a0cdc85de2bc939a3c6f9c601ef) regardless :)